src/python_testing/TC_AVANALYTestBase.py

```
@@ -46,3 +47,66 @@ async def check_avanaly_attribute(self, attribute, expected_value, endpoint):
         value = await self.read_avanaly_attribute_expect_success(endpoint=endpoint, attribute=attribute)
         asserts.assert_equal(value, expected_value,
                              f"Unexpected '{attribute}' value - expected {expected_value}, was {value}")
+
+    async def send_enable_context_triggers_command(self, endpoint, context_triggers, expected_status: Status = Status.Success):
+        try:
+            await self.send_single_cmd(cmd=Clusters.AvAnalysis.Commands.EnableContextTriggers(
```

Pylint (4.0.7)

[error] Instance of 'AVANALYTestBase' has no 'send_single_cmd' member

(E1101)